bazel/grpc_extra_deps.bzl

```diff
@@ -35,6 +35,6 @@ def grpc_extra_deps():
     go_rules_dependencies()
     go_register_toolchains()
 
-    apple_rules_dependencies()
+    apple_rules_dependencies(ignore_version_differences)
```

Comment:
Nit: Supply this as a keyword argument, rather than a positional argument. It's not likely, but it's possible that someone in the `rules_apple` repo adds a new keyword argument to the *beginning* of the list rather than the end. That would technically break API, but it wouldn't be unprecedented.

Reply:
That's a good point! I changed it!